dist/crunker.DESC.JSON

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I don't see why this file (or the `dist` directory as a whole) should be part of the repository?